pkg/apis/cluster/types.go

```diff
@@ -44,8 +44,8 @@ type Cluster struct {
 // ClusterSpec defines the desired state of a member cluster.
 type ClusterSpec struct {
 	// ID is the unique identifier for the cluster.
-	// It is different from the object uid(.metadata.uid) and typically collected automatically
-	// from member cluster during the progress of registration.
+	// It is different from the object uid(.metadata.uid) and is typically collected automatically
+	// from member clusters during the process of registration.
```

Comment:
Each ID corresponds to a cluster.  I‘m not sure whether to use singular or plural here.

Reply:
I think clusters is right in this case, but I could rephrase it as "from each member cluster" to make it more clear I think.